From baed1fedba161d7db89636a417751891831c432a Mon Sep 17 00:00:00 2001
From: Felipe Sateler <fsateler@users.noreply.github.com>
Date: Mon, 10 Oct 2016 10:40:05 -0300
Subject: [PATCH] login: drop fedora-specific PAM config, add note to
 DISTRO_PORTING (#4314)

It is impossible to ship a fully generic PAM configuration upstream.
Therefore, ship a minimal configuration with the systemd --user requirements,
and add a note to DISTRO_PORTING documenting this.

Fixes #4284
---
 src/login/systemd-user.m4 | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git b/src/login/systemd-user.m4 a/src/login/systemd-user.m4
index fe38b24fef..f188a8e548 100644
--- b/src/login/systemd-user.m4
+++ a/src/login/systemd-user.m4
@@ -2,9 +2,11 @@
 #
 # Used by systemd --user instances.
 
+account  include system-auth
+
 m4_ifdef(`HAVE_SELINUX',
 session  required pam_selinux.so close
 session  required pam_selinux.so nottys open
 )m4_dnl
 session  required pam_loginuid.so
-session optional pam_systemd.so
+session  include system-auth
-- 
2.9.0

